test/spline_layer_test.jl

```diff
@@ -32,14 +32,31 @@ function run_test(f, layer, atol)
     return sum(abs.(data_validate_pred.-data_validate_fn))/length(data_validate_fn) < atol
 end
 
-##test 01: affine function, Chebyshev and Polynomial basis
+ZygoteRules.@adjoint ZygoteRules.literal_getproperty(A::Tridiagonal, ::Val{:dl}) = A.dl,y -> Tridiagonal(dl,zeros(length(d)),zeros(length(du)),)
```

Comment:
put this in the library and we'll do a patch release when Zygote adds it.

Reply:
Ok.